manifests/interface/aws/secrets-manager/storage/install.yaml

```diff
@@ -0,0 +1,43 @@
+ocfVersion: 0.0.1
+revision: 0.1.0
+kind: Interface
+metadata:
+  prefix: "cap.interface.aws.secrets-manager.storage"
+  name: "install"
+  displayName: "Install"
+  description: "Installs AWS Secrets Manager storage"
+  documentationURL: https://capact.io/docs
+  supportURL: https://aws.amazon.com/secrets-manager
+  iconURL: https://storage.googleapis.com/dashboard-icons/aws.svg
+  maintainers:
+    - email: team-dev@capact.io
+      name: Capact Dev Team
+      url: https://capact.io
+
+spec:
+  input:
+    typeInstances: {}
+    parameters:
+      input-parameters:
+        jsonSchema:
+          value: |-
+            {
+              "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Question to the reviewer: I'm not sure whether the region should reside in additional input, or input on the interface. What do you think?

Reply:
Good question but to not complicate things, I would keep it here as it is more consistent with the helm values.